Transmission config flow

engrbm87 opened the pull request:
**Description:**
Configure Transmission through config flow.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26434

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

engrbm87 committed:
Transmission config flow

Configure Transmission using config flow.

klaasnicolaas commented:
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

MartinHjelmare commented on source/_components/transmission.markdown:
```diff
@@ -31,6 +31,8 @@ If everything is set up correctly, the details will show up in the frontend.
 
 ## Configuration
 
+Home Assistant offers UniFi integration through **Configuration** -> **Integrations** -> **Transmission**. For legacy support old transmission configuration is imported and set up as new integration.
```
```suggestion
Set up the integration through **Configuration** -> **Integrations** -> **Transmission**. For legacy support old transmission configuration is imported and set up as new integration.
```

frenck replied:
Applied suggestion.

frenck committed:
:pencil2: Tweak

engrbm87 commented:
I think we need to edit the description related to the `monitored_conditions` as they are no longer configured through configuration.yaml or imported.

frenck commented:
@engrbm87 It can simply be removed, right?

engrbm87 commented:
> 
> 
> @engrbm87 It can simply be removed, right?

Yes we can remove it. We can mention that the following sensors will be automatically added. And we need to add the start/stop all torrents switch which wasn't there before.

frenck commented:
Could you please update the documentation accordingly? Thanks :+1:

engrbm87 committed:
Update transmission.markdown

Removed monitored conditions and updated list of entities added by the integration

frenck reviewed (approved):
Thanks for the adjustments @engrbm87! 👍
